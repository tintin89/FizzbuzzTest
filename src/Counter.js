import React from 'react'

function Counter() {
    return (
        <div className="counter">
          <h2>0</h2>
        </div>
    )
}

export default Counter
