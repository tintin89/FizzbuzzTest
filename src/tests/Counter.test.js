import React from 'react';
import { shallow } from 'enzyme';
import Counter from '../Counter';


describe('Counter unit tests',()=>{
    let wrapper = shallow(<Counter/>);

    test('must show correctly the component Counter',()=>{

        expect(wrapper).toMatchSnapshot();
    })
})