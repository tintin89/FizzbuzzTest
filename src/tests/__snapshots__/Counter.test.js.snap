// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter unit tests test1,must show correctly the component Counter 1`] = `ShallowWrapper {}`;
